SLHCUpgradeSimulations/Geometry/test/phase2_digi_reco_pixelntuple_cfg.py

```diff
@@ -114,9 +113,9 @@
 # process.schedule imported from cff in HLTrigger.Configuration
 process.schedule.insert(0, process.digitisation_step)
 process.schedule.insert(1, process.L1simulation_step)
-process.schedule.insert(2, process.L1TrackTrigger_step)
-process.schedule.insert(3, process.digi2raw_step)
-process.schedule.extend([process.raw2digi_step,process.L1Reco_step,process.reconstruction_step,process.user_step,process.endjob_step])
+#process.schedule.insert(2, process.L1TrackTrigger_step)
+process.schedule.insert(2, process.digi2raw_step)
+process.schedule.extend([process.raw2digi_step,process.reconstruction_step,process.user_step,process.endjob_step])
```

Comment:
please leave a comment to remind the L1TT step needs to be re-inserted

Reply:
Done.

Reply:
@srimanob are you supposed to put back the L1Reco step eventually?
If so, the comment that was asked for in order to remind it is missing...

Reply:
there's no need to remove the `L1TrackTrigger` anymore after changing the input file.
The removal of the `L1Reco` instead is fine, as documented at https://github.com/cms-sw/cmssw/issues/33012.
So the current status of the PR is fine!